src/firewall/varktables/types.rs

```diff
@@ -200,6 +201,8 @@ pub fn get_network_chains<'a>(
     let mut chains = Vec::new();
     let prefixed_network_hash_name = format!("{}-{}", "NETAVARK", network_hash_name);
 
+    let dns_port: i64 = 53;
```

Comment:
we should not hardocde that, you can add the dns_port as parameter to the function and the use that one instead as user could have changed it.